src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -2321,6 +2321,7 @@ class ColumnConstraintSegment(BaseSegment):
                 OneOf(
                     Ref("LiteralGrammar"),
                     Ref("FunctionSegment"),
+                    Ref("BareFunctionSegment"),
                     # ?? Ref('IntervalExpressionSegment')
```

Comment:
Not related to the PR, but should we remove this line?

Reply:
Yeah let’s do it. Looks like it was added here for BigQuery but didn’t need to be in ANSI: https://github.com/sqlfluff/sqlfluff/pull/221

```suggestion
```